jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -373,10 +373,6 @@ var module = module;
           location: extraInfo['bzr-url']
         }
       };
-      if (meta['charm-related']) {
-        this._lowerCaseKeys(meta['charm-related'], meta['charm-related']);
-        processed.relatedCharms = meta['charm-related'];
```

Comment:
Did you forget to commit a file? In charm.js there are a number of methods which deal with the `relatedCharms` property.